guides/01_getting-started/01_quickstart.md

```diff
@@ -1,136 +1,114 @@
 
-$demo_hello_world
+## Building Your First Demo
 
-When developing locally, if you want to run the code as a Python script, you can use the Gradio CLI to launch the application **in reload mode**, which will provide seamless and fast development. Learn more about reloading in the [Auto-Reloading Guide](https://gradio.app/developing-faster-with-reload-mode/).
+You can run Gradio in your favorite code editor, Jupyter notebook, [Google Colab](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing), or anywhere else you write Python. Let's write our first Gradio app:
 
-```bash
-gradio app.py
-```
 
-Note: you can also do `python app.py`, but it won't provide the automatic reload mechanism.
+$code_hello_world
 
-## The `Interface` Class
 
-You'll notice that in order to make the demo, we created a `gr.Interface`. This `Interface` class can wrap any Python function with a user interface. In the example above, we saw a simple text-based function, but the function could be anything from music generator to a tax calculator to the prediction function of a pretrained machine learning model.
+Tip: We shorten the imported name from <code>gradio</code> to <code>gr</code> for better readability of code. This is a widely adopted convention that you should follow so that anyone working with your code can easily understand it. 
 
-The core `Interface` class is initialized with three required parameters:
+Now, run your code. If you've written the Python code in a file named, for example, `app.py`, then you would run `python app.py` from the terminal.
 
-- `fn`: the function to wrap a UI around
-- `inputs`: which component(s) to use for the input (e.g. `"text"`, `"image"` or `"audio"`)
-- `outputs`: which component(s) to use for the output (e.g. `"text"`, `"image"` or `"label"`)
+The demo below will open in a browser on [http://localhost:7860](http://localhost:7860) if running from a file. If you are running within a notebook, the demo will appear embedded within the notebook.
 
-Let's take a closer look at these components used to provide input and output.
+$demo_hello_world
 
-## Components Attributes
+Type your name in the textbox on the left, and then press the Submit button. You should see a friendly greeting on the right.
 
-We saw some simple `Textbox` components in the previous examples, but what if you want to change how the UI components look or behave?
+Tip: When developing locally, you can run your Gradio app in <strong>hot reload mode</strong>, which automatically reloads the Gradio app whenever you make changes to the file. To do this, simply type in <code>gradio</code> before the name of the file instead of <code>python</code>. In the example above, you would type: `gradio app.py` in your terminal. Learn more about hot reloading in the <a href="/guides/09_other-tutorials/developing-faster-with-reload-mode.md">Hot Reloading Guide</a>.
```

Comment:
```suggestion
Tip: When developing locally, you can run your Gradio app in <strong>hot reload mode</strong>, which automatically reloads the Gradio app whenever you make changes to the file. To do this, simply type in <code>gradio</code> before the name of the file instead of <code>python</code>. In the example above, you would type: `gradio app.py` in your terminal. Learn more about hot reloading in the <a href="/guides/developing-faster-with-reload-mode">Hot Reloading Guide</a>.
```

Reply:
Hmm I'm not sure if this will work when clicked from the readme. We'll need this to work on the guides though. Do you have any suggestions what to do here?